envd/api/__init__.py

```diff
@@ -42,11 +42,12 @@ def shell(name: str):
     """
 
 
-def run(commands: str):
+def run(commands: str, mount_host: bool = False):
     """Execute command
 
     Args:
         commands (str): command to run during the building process
+        mount_host (bool): mount the host directory
```

Comment:
Better to explicitly say this again since it's a breaking change
```suggestion
        mount_host (bool): mount the host directory. Default is False.
```